pypeit/core/wavecal/templates.py

```diff
@@ -159,7 +159,7 @@ def pypeit_arcspec(in_file, slit):
     wv_vac = utils.func_val(iwv_calib['fitc'], x/iwv_calib['xnorm'], iwv_calib['function'],
                            minx=iwv_calib['fmin'], maxx=iwv_calib['fmax'])
     # Return
-    return wv_vac, np.array(iwv_calib['spec'])
+    return wv_vac, np.array(iwv_calib['spec']).flatten()  # JXP added flatten on 2019-11-09
```

Comment:
Can you explain why this was necessary in a comment?

Reply:
I honestly don't recall.  It was 4 months ago.